controllers/validation/account_validation_controller.go

```diff
@@ -439,9 +448,99 @@ func (r *AccountValidationReconciler) Reconcile(ctx context.Context, request ctr
 			}
 		}
 	}
+
+	// check if account belongs to accountpool
+	if !account.IsBYOC() {
+		awsRegion := config.GetDefaultRegion()
+		awsSetupClient, err := r.awsClientBuilder.GetClient(controllerName, r.Client, awsclient.NewAwsClientInput{
+			SecretName: utils.AwsSecretName,
+			NameSpace:  awsv1alpha1.AccountCrNamespace,
+			AwsRegion:  awsRegion,
+		})
+		if err != nil {
+			connErr := fmt.Sprintf("unable to connect to default region %s", awsRegion)
+			reqLogger.Error(err, connErr)
+			return ctrl.Result{}, err
+		}
+
+		if account.Spec.RegionalServiceQuotas != nil {
+			if account.Status.RegionalServiceQuotas == nil {
+				err = acct.SetCurrentAccountServiceQuotas(reqLogger, r.awsClientBuilder, awsSetupClient, &account, r.Client)
+				if err != nil {
+					reqLogger.Error(err, "failed to set account service quotas")
+					return reconcile.Result{}, err
+				}
+				err := r.statusUpdate(&account)
+				if err != nil {
+					return reconcile.Result{}, err
+				}
+
+				return reconcile.Result{RequeueAfter: 30 * time.Second}, nil
+			} else {
+				_, err = r.getServiceQuotaRequest(reqLogger, &account, awsSetupClient, shardName)
+				if err != nil {
+					reqLogger.Error(err, "failed to get account service quota")
+					return reconcile.Result{RequeueAfter: 30 * time.Second}, err
+				}
+				for _, quotas := range account.Status.RegionalServiceQuotas {
+					for _, quota := range quotas {
+						if quota.Status == "TODO" || quota.Status == "IN_PROGRESS" {
+							return reconcile.Result{RequeueAfter: 10 * time.Minute}, nil
+						}
+					}
+				}
+
+			}
+		}
+	}
 	return utils.DoNotRequeue()
 }
 
+func (r *AccountValidationReconciler) getServiceQuotaRequest(reqLogger logr.Logger, currentAcctInstance *awsv1alpha1.Account, awsSetupClient awsclient.Client, shardName string) (reconcile.Result, error) {
+	if currentAcctInstance.HasOpenQuotaIncreaseRequests() {
+		switch utils.DetectDevMode {
+		case utils.DevModeProduction:
+			// First we get all request we need to get a status update on:
+			// - Requests that are not yet open on the AWS side
+			// - Requests that are open but not yet completed
+			reqLogger.Info("Look here???")
+			currentInFlightCount, inFlightQuotaRequests := currentAcctInstance.GetQuotaRequestsByStatus(awsv1alpha1.ServiceRequestInProgress)
```

Comment:
This whole code block seems to be a copy of the https://github.com/reedcort/aws-account-operator/blob/3e2296fe1b5c73a6a7ed93facda82cdb0c671242/controllers/account/account_controller.go#L599-L637 and might be better served to be extracted somewhere.